src/Controller/API/MeshDescriptors.php

```diff
@@ -30,7 +30,7 @@ public function __construct(MeshDescriptorManager $manager)
 
     /**
      * Handle the special 'q' parameter
-     * @Route("/", methods={"GET"})
+     * @Route("", methods={"GET"})
```

Comment:
No, when it has `/` then something like `api/v2/meshdescriptors?sortBy=id` gets redirected to `api/v2/meshdescriptors/?sortBy=id` so each request ends up generating a `302` response instead of data. Removing it seems to fix that.